lib/rubygems/ext/rake_builder.rb

```diff
@@ -20,7 +20,7 @@ def self.build(extension, dest_path, results, args=[], lib_dir=nil)
       rake = rake.shellsplit
     else
       begin
-        rake = [Gem.ruby, "-rrubygems", Gem.bin_path('rake', 'rake')]
+        rake = [Gem.ruby, "-I#{File.expand_path("..", __dir__)}", "-rrubygems", Gem.bin_path('rake', 'rake')]
```

Comment:
I don't think this change is necessary, although I guess it doesn't harm. I just don't like it polluting an unrelated PR.